packages/api/scripts/cmds/services.js

```diff
@@ -5,12 +5,31 @@ import { MINIO_API_PORT } from './minio.js'
 import { isPortReachable } from '../utils.js'
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url))
-const composePath = path.join(__dirname, '../../docker/docker-compose.yml')
+const composeDir = path.join(__dirname, '../../docker')
+const composeFiles = [
+  'docker-compose.yml',
+  'docker-compose-local-ports.yml',
+  'docker-compose-volumes.yml',
+]
+
+const composeFileArgs = composeFiles.flatMap((filename) => [
+  '--file',
+  path.join(composeDir, filename),
+])
 
 const PG_PORT = 3000
 const PGRST_PORT = 5432
 const CLUSTER_PORT = 9094
```

Comment:
do we need this ?

Reply:
I think so, since this PR is still using fixed ports for dev (see other comment).